#
# use port as before calling to config_write()
# fix has been submitted to upstream:
#	https://github.com/proftpd/proftpd/pull/1524

--- a/tests/t/lib/ProFTPD/Tests/Signals/HUP.pm
+++ b/tests/t/lib/ProFTPD/Tests/Signals/HUP.pm
@@ -89,7 +89,7 @@ sub hup_daemon_ok {
 
   # Now change the config a little, and send the HUP signal
   $config->{ServerIdent} = 'on bar';
-  ($port, $config_user, $config_group) = config_write($config_file, $config);
+  config_write($config_file, $config);
   server_restart($pid_file);
   sleep(2);
 
